prow/plugins/trigger/pr.go

```diff
@@ -49,7 +49,26 @@ func handlePR(c client, trustedOrg string, pr github.PullRequestEvent) error {
 				return fmt.Errorf("could not welcome non-org member %q: %v", author, err)
 			}
 		}
-	case github.PullRequestActionReopened, github.PullRequestActionSynchronize:
+	case github.PullRequestActionReopened:
+		// When a PR is reopened, check that the user is in the org for now. If so, run the building.
+		orgMember, err := c.GitHubClient.IsMember(trustedOrg, author)
```

Comment:
Use `trustedPullRequest` instead of `IsMember` so that if a PR has been marked trusted with `/ok-to-test` it will be rebuilt when the PR is reopened.

Reply:
If we expect not only the new members' reopened PRs but also all of the trusted reopened PRs to be rebuilt once they are reopened, `trustedPullRequest ` makes sense to me. Will address this when I'm back to the office in the morning :)